go.mod

```diff
@@ -30,6 +30,12 @@ require (
 
 replace github.com/btcsuite/btcwallet/walletdb => ./walletdb
 
+replace github.com/btcsuite/btcd => github.com/guggero/btcd v0.20.1-beta.0.20220214155024-51fd177d32c0
```

Comment:
What's behind the fork of `btcd` and `btcec/v2`?  Do you have link to diff vs upstream?

Reply:
Just a few fixes I found while end-to-end testing https://github.com/btcsuite/btcd/pull/1787. Might not be necessary after the latest update on that PR.

Reply:
See https://github.com/guggero/btcd/tree/schnorr-sig, this is the current diff (which will hopefully be integrated into btcd#1787)

Reply:
How about opening a new PR for that (targeting https://github.com/Roasbeef/btcd/tree/taproot-impl until it is merged)? the branch/PR is already quite large, but it would be great to be able to review that separately already IMO :)

Reply:
I think his fork should now be more or less identical to my main PR.

Reply:
All my changes were either incorporated into https://github.com/btcsuite/btcd/pull/1787 or moved to another repo. So there is no need for another branch/PR anymore.